lib/jirawatch/cli/track.rb

```diff
@@ -6,55 +6,84 @@ class Track < Dry::CLI::Command
       include Jirawatch::CLI::AuthenticatedCommand
 
       argument :issue_key, required: true, desc: "Issue key that you want to track time for"
-      option :started_at, aliases: ["-t"], desc: "Specify when you started working on this issue with a HH:mm format"
+      option :time_started_at, aliases: ["-t"], desc: "Specify when you started working on this issue with a HH:mm format"
       option :worklog_message, aliases: ["-m"], desc: "Specify a work log message"
 
       def call(issue_key:, **options)
         @jira_client.Issue.find(issue_key) # Fails if issue doesn't exist
 
+        time_started_at = Time.parse(options.fetch(:time_started_at, Time.now.to_s))
+        time_paused_at = 0
+        time_restarted_at = time_started_at
+        partial_time_spent = 0
+
         puts "Logging time for #{issue_key}..."
-        started_at = Time.parse(options.fetch(:started_at, Time.now.to_s))
-        puts started_at
+        puts time_started_at
+        puts
+        puts "Press CTRL-P to pause/restart the timer"
```

Comment:
`Hit CTRL-P to pause/restart tracking`

Reply:
I would keep `press` : `Press CTRL-P to pause/restart tracking time`